### PR TITLE
To-Do List 

### PR DESCRIPTION
**Pull request by fobadara**
To-Do list: Structure

In this version I fulfilled the following:

:green_circle: I made use of the MVC design pattern and Object-oriented Programming principles

 I also fulfilled the following requirements:

:green_circle:  I have an array that takes objects with three keys.

:green_circle: It populates the HTML file in the index order.

:green_circle: All files are loaded with webpack.

Please ignore all extra functionality I was practicing some tools.

### CODE REVIEW

File: src/model.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
+export default class Model {
+  constructor() {
+    this.tasks = JSON.parse(localStorage.getItem('todos')) || [];
+  }
+
+  setStorage = (todos) => {
+    // this.tasks.forEach(element => {
+    localStorage.setItem('todos', JSON.stringify(todos));
+    // });
```
Comment (de-scholar) [inline src/model.js:9]: - [ ] [Optional] I would advise you remove commented codes in order to make your code more clean

File: server.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    publicPath: config.output.publicPath,
+  }),
+);
+
+// Serve the files on port 3000.
+app.listen(3000, () => {
+  /* eslint-disable */
+  console.log('Example app listening on port 3000!\n');
+   /* eslint-enable */
+});
```
Comment (de-scholar) [inline server.js:24]: - [ ] [Optional] Great on doing this, but I think webpack server was sufficient, it was not needed to write your own server. But I like this initiative of going beyond the requirements, just for learning purpose

**Review by de-scholar — APPROVED** [review]
Hi @fobadara , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/model.js (hunk 1 above)
Comment (fobadara) [reply src/model.js:9]: I will make sure I do that tomorrow. Thank you

**Pull request merged**